public/Scripts/Mapa/mapa_atacar.php

```diff
@@ -149,8 +149,15 @@
         $protector->exit_error("Um dos requisitos para atacar esse alvo não está cumprido.");
     }
 }
-
-
+$verifica = $connection->run("SELECT unix_timestamp(TIMEDIFF(current_timestamp, horario)) AS duracao 
+                              FROM tb_combate_log 
+                              WHERE (id_1 = ? OR id_2 = ?)
```

Comment:
essa logica ta errada, deveria ter (id_1 = combatente_a AND id_2 = combatente_b) OR (id_1 = combatente_b AND id_2 = combatente_a)

Reply:
nao entendi essa